system/boot/ix86/isoboot/suse-linuxrc

```diff
@@ -210,7 +213,7 @@ if [ -z "$UNIONFS_CONFIG" ] && [ -z "$COMBINED_IMAGE" ];then
         systemException "Failed to mount RW root filesystem" "reboot"
     fi
     cdopt=$(CDMountOption $biosBootDevice)
-    mkdir -p $LIVECD && eval mount $cdopt $biosBootDevice $LIVECD
+    mountLivecd
```

Comment:
the name is misleading, maybe better mountOrCopyLiveCD or embedLiveCD or integrateLiveCD